### PR TITLE
feature: spec add supportedErdaVesions field

### PR DESCRIPTION
**Pull request by sfwn**
Related Erda PR: https://github.com/erda-project/erda/pull/616

### CODE REVIEW

**Comment by sfwn** [comment]
https://github.com/erda-project/erda-actions/pull/55

**Review by Effet — CHANGES_REQUESTED** [review]
missing supportedErdaVersions

File: actions/docker-push/1.0/spec.yml [hunk 1]
```diff
@@ -5,5 +5,5 @@ displayName: 容器推送
 category: custom_task
 params:
 public: true
-supportedVersions:
+supportedVersions: # Deprecated. Please use supportedErdaVersions instead.
```
Comment (Effet) [inline actions/docker-push/1.0/spec.yml:8]: missing supportedErdaVersions

File: actions/java-agent/1.0/spec.yml [hunk 2]
```diff
@@ -3,5 +3,5 @@ version: '1.0'
 type: action
 desc: 监控java agent注入
 public: false
-supportedVersions:
+supportedVersions: # Deprecated. Please use supportedErdaVersions instead.
```
Comment (Effet) [inline actions/java-agent/1.0/spec.yml:6]: missing supportedErdaVersions

**Review by Effet — CHANGES_REQUESTED** [review]

**Review by Effet — APPROVED** [review]

**Pull request merged**